diff --git a/fetch.c b/fetch.c
index 2ec045f..5716166 100644
--- a/fetch.c
+++ b/fetch.c
@@ -28,10 +28,16 @@
 
 
 /* include system configuration before all else. */
+#ifdef __MVS__
+#define fetch fetch_stdlib
+#endif
 #include "autoconf.h"
 
 #include "gdbmdefs.h"
 #include "extern.h"
+#ifdef __MVS__
+#undef fetch
+#endif
 
 
 /* Look up a given KEY and return the information associated with that KEY.
